src/FlightMap/Widgets/QGCInstrumentWidget.qml

```diff
@@ -55,6 +59,111 @@ ColumnLayout {
             vehicle:                globals.activeVehicle
             anchors.verticalCenter: parent.verticalCenter
         }
+    }  
+
+    enum WinchCommands {
+        RELEASE = 0,
+        DELIVER = 4,
+        RETRACT = 6
+    }
+    
+    property int _currentCommand:   -1
+    property int _WinchNodeID:      42
+    property int _MAVLink_CMD_ID:   42600
```

Comment:
1. Inconsistent casing
2. "_currentCommand" is way too generic.
3. Is it a good idea to keep the status of Slider as a variable outside the scope? IDK if variables here become global, but would it be possible to just define that variable on the Slider object, and let the QGCButtons modify it?
4. It's unclear why _WinchNodeID is 42, and why _MAVLink_CMD_ID is 42600. Add comments for such specific values
5. Consider using null instead of -1 for uninitialized, it makes it more obvious what's going on.
6. _MAVLink_CMD_ID is also too generic a name, granted that it's particular to the winch. Also, why does this need to be defined as a variable here at all? It doesn't change. I'd guess this was available in the MAVLink library, or possibly imported from MavCmdInfoCommon.json somehow

Reply:
1. Improved casing to match QGC QML standards (although QGC is not 100% consistent, but I tried to match other widgets)
2. Renamed to reflect that it is a winching command
3. QML is dynamically scoped, but I moved current winch command to slider.
4. Added comments to the commands and components.
5. -1 -> null
6. Removed entirely because it doesn't change, but put in two comments to explain what they are and where they come from.